src/cleo/commands/completions_command.py

```diff
@@ -156,13 +157,14 @@ def render_bash(self) -> str:
         for cmd in sorted(self.application.all().values(), key=lambda c: c.name or ""):
             if cmd.hidden or not cmd.enabled or not cmd.name:
                 continue
-            cmds.append(cmd.name)
+            command_name = shell_quote(cmd.name) if " " in cmd.name else cmd.name
```

Comment:
Does it hurt to unconditionally quote the command name?

Reply:
Don't think so. This is mostly here for me to remind me about this being an issue when refactoring completions.